src/components/Tooltip/Tooltip.tsx

```diff
@@ -166,7 +176,9 @@ export const Tooltip = ({
     ) : (
         <BlueprintTooltip
             lazy={true}
-            hoverOpenDelay={hoverOpenDelay - swapDelayTime}
+            hoverOpenDelay={
+                swapDelayTime > 0 && hoverOpenDelay > swapDelayTime ? hoverOpenDelay - swapDelayTime : hoverOpenDelay
```

Comment:
If the hover open delay is greater than the swap delay time, e.g. hover delay 500ms and swap delay 300ms, then the actual hoverOpenDelay value would be 200ms and not 500ms for the subsequent hover events. This only makes sense for the hover event with the actual swap.